src/screen/ConnectionsList/index.tsx

```diff
@@ -12,6 +13,12 @@ const ConnectionsList: React.FC = () => {
   const [connections] = useRecoilState(connectionsState)
   const [isCreateModalOpen, toggleCreateModalOpen] = useToggle(false)
 
+  useEffect(() => {
+    ipcRenderer.on('newConnection', function () {
```

Comment:
Maybe we should return a function from this useEffect to unregister the ipcRenderer event listener.